samtranslator/region_configuration.py

```diff
@@ -18,4 +18,9 @@ def is_apigw_edge_configuration_supported(cls):
         :return: True, if API Gateway does not support Edge configuration
         """
 
-        return ArnGenerator.get_partition_name() not in [cls.partitions["govcloud"], cls.partitions["china"]]
+        return ArnGenerator.get_partition_name() not in [
```

Comment:
minor: Looks like the list can be replaced by cls.partitions.values(), so we don't change two places.

Reply:
Isn't it cls.partitions.keys()?

Reply:
+1 for using `cls.partitions.values()`

it should be `cls.partitions.values()` instead of `cls.partitions.keys()`, because for example we are checking whether `"aws-iso"` is in the list, but not `"iso"`.

Reply:
Yes indeed, I removed my comment :)

Reply:
Good catch. Not sure why this is defined this way in general. Will simplify.

Reply:
There was no need for the dictionary that I could find. So I just pushed everything into the `in` statement.